### PR TITLE
ignore trailing slashes for flag --exclude and improve usage info #20

### PR DESCRIPTION
**Pull request by martin-schlossarek**
`--exclude /path/to/exclude/` now excludes `/path/to/exclude`

also `--help` now mentions that `--exclude` can be used multiple times.

### CODE REVIEW

File: scanner/main.go [hunk 1]
```diff
@@ -62,6 +62,14 @@ func handleJar(path string, ra io.ReaderAt, sz int64) {
 	}
 }
 
+func trimTrailingSlashes(path string) string {
```
Comment (hillu) [inline scanner/main.go:65]: I think that `path/filepath.Clean()` should do the trick here.

File: scanner/main.go [hunk 2]
```diff
@@ -75,7 +83,7 @@ func (flags *excludeFlags) Set(value string) error {
 
 func (flags excludeFlags) Has(path string) bool {
 	for _, exclude := range flags {
-		if path == exclude {
+		if trimTrailingSlashes(path) == trimTrailingSlashes(exclude) {
```
Comment (hillu) [inline scanner/main.go:86]: Please clean up the path in `Set()`

**Review by hillu — CHANGES_REQUESTED** [review]

File: scanner/main.go (hunk 1 above)
Comment (martin-schlossarek) [reply scanner/main.go]: Thanks for the hint

File: scanner/main.go (hunk 2 above)
Comment (martin-schlossarek) [reply scanner/main.go]: Done, but I also had to add a `filepath.Clean` in the `filepath.Walk` function

**Pull request merged**